dags/common/utils.py

```diff
@@ -291,6 +290,58 @@ def article_type_filter(article_type: str, types: list) -> bool:
         return False
     return article_type.lower() in set(t.lower() for t in types)
 
+def title_filter(case_report: dict) -> bool:
+    """Checks to see if title shows a cardiology case_report
+    Args:
+        case_report (dict): json file
+        terms (list): list of cardiology terms
+    Returns:
+        bool: returns whether title is cardiology-related
+    """
+    title = case_report.get("title")
+    terms = ["heart", "cardiology", "heartrhythm", "cardiovascular", "heart rhythm", "cardio"]
+    temp = []
+    if isinstance(title, str):
+        temp.extend([x.upper() for x in terms if x.lower() in title.lower()])
+    if len(temp) > 0:
+        return True
+    else:
+        return False
+
+def text_filter(case_report: dict) -> bool:
+    """Checks to see if text is a cardiology case report
+    Args:
+        case_report (dict): json file
+        terms (list): list of cardiology terms
+    Returns:
+        bool: returns whether text is cardiology-related
+    """
+    text = case_report.get("text")
+    terms = ["heart", "cardiology", "heartrhythm", "cardiovascular", "heart rhythm", "cardio"]
+    temp = []
+    if isinstance(text, str):
+        temp.extend([x.upper() for x in terms if x.lower() in text.lower()])
+    if len(temp) > 0:
+        return True
+    else:
+        return False
+
+def cardio_filter(case_report: dict) -> bool:
+    """Checks to see if document is a cardiology case_report
+    Args:
+        case_report (dict): pubmed json file
+        terms (list): list of cardiology terms
+    Returns:
+        bool: returns whether document is cardiology-related
+    """
+    journal_title = case_report.get("journal")
+    terms = ["heart", "cardiology", "heartrhythm", "cardiovascular", "heart rhythm", "cardio", "JACC"]
+    temp = []
```

Comment:
I feel like this could be implemented differently. It should return true as soon as it finds a match.